dev/devicelab/manifest.yaml

```diff
@@ -635,6 +635,12 @@ tasks:
     required_agent_capabilities: ["mac/android"]
     flaky: true
 
+  smoke_catalina_start_up:
+    description: >
```

Comment:
Should be marked as flaky

Reply:
How come?

Reply:
Every new test/host should be marked as flaky for a duration of time to avoid infra setup issues from effectively closing the tree

Reply:
sgtm.  Is that documented anywhere?

Reply:
Done.